.gitpod.yml

```diff
@@ -1,6 +1,7 @@
 tasks:
   - name: Setup
     init: |
+      ln --symbolic contrib/ide/vscode .vscode
```

Comment:
A question, should it use `cp` instead? In case the user would modify their local `.vscode`.